src/common/components/DateInfo.tsx

```diff
@@ -22,8 +26,45 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const DateInfo = ({ date }: DateInfoProps): JSX.Element => {
+const DateInfo = ({ resource }: DateInfoProps): JSX.Element => {
   const classes = useStyles();
+  const date = useMemo(() => {
+    const dateOrPeriod = getResourceDateOrPeriod(resource);
+    if (typeof dateOrPeriod === "string") {
+      return DateTime.fromISO(dateOrPeriod).toLocaleString(
+        {
+          day: "numeric",
+          month: "long",
+          year: "numeric",
+        },
+        {
+          locale: navigator.language,
```

Comment:
for my culture: is this always defined ?

Reply:
I guess so, this is supported by all browsers : 

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language